packages/addons-v5/test/commands/addons/open.js

```diff
@@ -8,6 +8,34 @@ let cmd = commands.find((c) => c.topic === 'addons' && c.command === 'open')
 describe('addons:open', function () {
   beforeEach(() => cli.mockConsole())
 
+  describe('testing sudo', function () {
+    beforeEach(() => {
+      process.env.HEROKU_SUDO = true
+    })
+
+    it('with sudo && sso.method is true', function () {
```

Comment:
Could you please update the it statements you added in this file to state what the expected result should be?